back/app/controllers/web_api/v1/project_custom_fields_controller.rb

```diff
@@ -27,11 +27,16 @@ def project
     @project ||= Project.find_by id: params[:project_id]
   end
 
-  def participation_context
-    @participation_context ||= determine_participation_context
+  def input_term
+    participation_context = ParticipationContextService.new.get_participation_context(project)
+    participation_context ? participation_context.input_term : project.input_term
```

Comment:
It doesn't seem possible to set the input term of the project for timeline projects. Shouldn't we have a different fallback when there is no current phase? For example the last previous phase (and the next one when there is none)?

Reply:
Alright, sounds good :)

Reply:
Never. The phase is always given when posting in a timeline project via the BO. That is exactly the reason why we introduced the menu with the phases. Note that when there is only 1 phase, the menu does not appear, but the same URL with the `phase_id` param is used.

Example: going to
http://localhost:3000/en/projects/ideation-and-native-survey/ideas/new?phase_id=2e9a5d35-8af5-42d1-982d-660b3fb9917f
triggers
http://localhost:3000/web_api/v1/phases/2e9a5d35-8af5-42d1-982d-660b3fb9917f/custom_fields/json_forms_schema